hummingbot/connector/gateway_in_flight_order.py

```diff
@@ -19,58 +16,24 @@ def __init__(self,
                  price: Decimal,
                  amount: Decimal,
                  creation_timestamp: float,
-                 gas_price: Decimal,
-                 initial_state: str = "PENDING_CREATE"):
+                 gas_price: Optional[Decimal] = Decimal("0"),
+                 initial_state: str = OrderState.PENDING_CREATE,
+                 is_approval: bool = False):
```

Comment:
```suggestion
```
I see `is_approval` isn't really used in this class. Let us clean it up here and in initializing calls.

Reply:
It is used in line 31

Reply:
I still see it's redundant. Your changes in `hummingbot/connector/gateway_EVM_AMM.py` already send the appropraite `initial_state`.

Reply:
Ahh I see what you mean. Ok